### DISCUSSION TITLE
Can't enter PAT for connecting cloud9 and github as a password

### DISCUSSION THREAD

**Discussion by ryucy-k** [community/community#23509] [New to GitHub]
<p>ec2-user:~/environment/test (master) $ git push origin master<br>
Username for ‘<a href="https://github.com/@@@@/test.git" rel="noopener nofollow ugc">https://github.com/@@@@/test.git</a>’: @@@@<br>
Password for ‘https://@@@@<span class="mention">@github.com</span>/@@@@/test.git’:</p>
<p>I could enter my username but I couldn’t enter the password(PAT).<br>
I couldn’t even enter any words in Password section.</p>
<p>Can anyone help me Please???</p>

**Comment by jsoref** [comment, marked as answer]
<p>Unix programs don’t tend to <em>show</em> how many characters are in your password, they consider that part of the security. Just trust that keys you type are being listened to by the program.</p>

**Comment by ryucy-k** [comment]
<p>Thank you so much!!<br>
problem solved!!</p>
